recipe/meta.yaml

```diff
@@ -30,12 +37,122 @@ requirements:
     - make
     - cmake
   host:
+    - cmake
     - libblas
     - liblapack
 
-test:
-  commands:
-    - openfast -h
+outputs:
+  - name: openfast
+    build:
+      script:
+        - cmake {{ cmake_args }} -S . -B build
+        - cmake --build build --target openfast --parallel
```

Comment:
You can't just build it, you have to install it. 

If you want to install just one target, maybe this can help: https://stackoverflow.com/questions/17164731/installing-only-one-target-and-its-dependencies-out-of-a-complex-project-with